test/PowerShellEditorServices.Test.Shared/Symbols/DSCFile.ps1

```diff
@@ -0,0 +1,4 @@
+﻿# This file represents a script with a DSC configuration
+configuration AConfiguration {
```

Comment:
@fflaten did you have DSC working? I had to disable the test entirely, the visitor was never getting called. Patrick and I thought it might be a module-import issue.

Reply:
AFAIK all tests passed (DSC only on Windows, skipped on rest). I know I had problems making it DSC work on Unix even after installing all the dependencies and I develop in devcontainer so might not have tested it manually.

I'm not against skipping it. Mostly cosmetic feature since you usually have one configuration per file (I assume). Isn't the `configuration` keyword also being removed in DSC v3?

Reply:
No worries and yeah, not super concerned about it right now.